includes/sections-menu-common.php

```diff
@@ -53,22 +53,19 @@ public static function format_shortcode_output( $content ) {
 	function section_menus_display_default( $selector ) {
 		ob_start();
 	?>
-		<nav class="navbar navbar-toggleable-md navbar-light bg-primary sections-menu">
+		<nav class="navbar navbar-toggleable-md navbar-light bg-primary sections-menu" style="box-shadow: 0 2px 32px rgba(0,0,0,.5);">
```

Comment:
We should probably add a stylesheet that sets the box-shadow on .sections-menu when .fixed-top is also applied instead of hard-coding it here (on the main site/students themes, we only show the box shadow once the bar is affixed)